### ISSUE TITLE
A lot of warnings for mo files.

### ISSUE THREAD

**Issue by JulienPalard** [roskakori/pygount#11]
I think that mo files should just be ignored as they are not plain text, while I'm getting loads of:

```
WARNING:pygount:cannot read ./git-clones/tendenci/tendenci/tendenci/locale/it/LC_MESSAGES/django.mo using encoding cp1252: 'charmap' codec can't decode byte 0x9d in position 48: character maps to <undefined>
```

**Comment by roskakori** [comment]
Technically pygments identifies `*.mo` as Modelica source code:

```python
>>> import pygments.lexers
>>> lexer = pygments.lexers.get_lexer_for_filename('some.mo')
>>> lexer.name
'Modelica'
```
Automatically excluding binary files from analysis in theory makes sense. However, detecting binaries is non trivial. The most common approach seems to be checking for 0 bytes as used by [Subversion](http://subversion.apache.org/faq.html#binary-files) and [gitattributes](https://git-scm.com/docs/gitattributes). The actual [code fom git](http://git.kernel.org/?p=git/git.git;a=blob;f=xdiff-interface.c;h=0e2c169227ad29b5bf546c6c1b97e1a1d8ed7409;hb=HEAD) is:

```c
#define FIRST_FEW_BYTES 8000
 int buffer_is_binary(const char *ptr, unsigned long size)
 {
         if (FIRST_FEW_BYTES < size)
                 size = FIRST_FEW_BYTES;
         return !!memchr(ptr, 0, size);
 }
```
In case of pygments it would make sense to treat files with headers for UTF-16 and UTF-32 as text despite plenty of 0 bytes in it.

**Comment by roskakori** [comment]
I implemented the proposed solution with v0.9 (check for BOM first, then for zero bytes within the initial 8K).

It's already available from PyPI so you can give it a try.
